docs/core-concepts/packages.md

```diff
@@ -0,0 +1,143 @@
+- [System.Collections](https://www.nuget.org/packages/System.Collections) - A set of (primarily) generic collections, including [List&lt;T&gt;](http://dotnet.github.io/api/System.Collections.GenericList'1) and [Dictionary&lt;K,V&gt;](http://dotnet.github.io/api/System.Collections.Generic.Dictionary'1,'2).
+- [System.Net.Http](https://www.nuget.org/packages/System.Net.Http) - A set of types for HTTP network communication, including [HttpClient](http://dotnet.github.io/api/System.Net.Http.HttpClient) and [HttpResponseMessage](http://dotnet.github.io/api/System.Net.Http.HttpResponseMessage).
+- [System.IO.FileSystem](https://www.nuget.org/packages/System.IO.FileSystem) - A set of types for reading and writing to local or networked disk-based storage, including File and Directory.
+- [System.Linq](https://www.nuget.org/packages/System.Linq) - A set types for querying objects, including Enumerable and [ILookup&lt;TKey, TElement&gt](http://dotnet.github.io/api/System.Linq.ILookup'1,'2');.
+- [System.Reflection](https://www.nuget.org/packages/System.Reflection) - A set of types for loading, inspecting and activating types, including Assembly, TypeInfo and MethodInfo.
+
+Packages are referenced in project.json. In the example below, the [System.Runtime](https://www.nuget.org/packages/System.Runtime/) package is referenced. 
+
+```json
+{
+  "dependencies": {
+    "System.Runtime": "4.1.0"
+  },
+  "frameworks": {
+    "netstandard1.5": {}
+  }
+}
+
+In most cases, you will not reference the lower-level .NET Core packages directly since you'll end up with too many packages to manage. Instead, you'll reference a metapackages.
+
+Metapackages
+============
+
+Metapackages are a NuGet package convention for describing a set of packages that are meaningful together. They represent this set of packages by making them dependencies. They can optionally establish a target framework for this set of packages by specifying a framework. 
+
+By referencing a metapackage, you are, in effect, adding a reference to each of its dependent packages as a single gesture. That means that all of the libraries in those packages (refs or libs) are available for Intellisense (or similar experience) and for publishing (libs only) your app. 
+
+There are advantages to using metapackages:
+
+- Provide a convenient user experience to reference a large set of fine-grained packages. 
+- Defines a set of packages (including specific versions) that are tested and work well together.
+
+Note: The 'lib' and 'ref' terms refer to folders in NuGet packages. 'ref' folders describe the public API of a package via assembly metadata. 'lib' folders contain the implementation of that public API for a given target framework. 
```

Comment:
This seems to presume that the reader is seeing "lib" and "ref" here in this doc somewhere nearby above, but I don't see them.